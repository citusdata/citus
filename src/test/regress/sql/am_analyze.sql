--
-- Test the ANALYZE command for cstore_fdw tables.
--

-- ANALYZE uncompressed table
ANALYZE contestant;
SELECT count(*) FROM pg_stats WHERE tablename='contestant';

-- ANALYZE compressed table
ANALYZE contestant_compressed;
SELECT count(*) FROM pg_stats WHERE tablename='contestant_compressed';
