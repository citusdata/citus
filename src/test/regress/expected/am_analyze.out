--
-- Test the ANALYZE command for cstore_fdw tables.
--
-- ANALYZE uncompressed table
ANALYZE contestant;
SELECT count(*) FROM pg_stats WHERE tablename='contestant';
 count
---------------------------------------------------------------------
     6
(1 row)

-- ANALYZE compressed table
ANALYZE contestant_compressed;
SELECT count(*) FROM pg_stats WHERE tablename='contestant_compressed';
 count
---------------------------------------------------------------------
     6
(1 row)

